### PR TITLE
Translate README.md into Chinese

### PR DESCRIPTION
**Pull request by xfq**
Partial fix for #194.

### CODE REVIEW

**Review by r12a — APPROVED** [review]

**Review by r12a — APPROVED** [review]

**Comment by r12a** [comment]
@xfq any reason we're not merging this yet?

**Comment by xfq** [comment]
@r12a I think it'd be good to request review before merging it.

/cc @w3c/chinese-layout-req-contributors

File: README.md [hunk 1]
```diff
@@ -18,28 +18,42 @@ Topics for discussion are suggested by [the gap-analysis template](https://w3c.g
...
 
-- [Chinese Layout Gap Analysis](https://w3c.github.io/clreq/gap-analysis/)
+- [Chinese Layout Gap Analysis | 中文支持差距分析](https://w3c.github.io/clreq/gap-analysis/)
 - [Requirements for Chinese Text Layout | 中文排版需求 (ED)](https://w3c.github.io/clreq/) ([WD](https://www.w3.org/TR/clreq/))
 
 ### Following 跟踪
 
 Rather than just 'Watch' this repository, it is best to subscribe to the [public-i18n-cjk](https://lists.w3.org/Archives/Public/public-i18n-cjk/) mailing list. That list is notified once a day (in digest form) about not only changes to this repository, but also about other W3C Working Group issues related to the Chinese writing systems. (Please use github issues rather than the mailing list to send feedback. The main purpose of the public-i18n-mongolian mailing list is for notifications; meeting minutes are also sent to public-i18n-cjk.)
 
+除了“Watch”此仓库，还可以订阅 [public-i18n-cjk](https://lists.w3.org/Archives/Public/public-i18n-cjk/) 邮件列表，该列表每天以摘要形式通知此仓库的活动以及其他W3C工作组中与中文相关的问题。（请使用 GitHub Issues 而不是邮件列表发送反馈，因为 public-i18n-cjk 邮件列表的主要作用是提醒邮件及发布会议纪要。）
```
Comment (upsuper) [inline README.md]: “相比于……，……会更好”应该更切合“Rather than..., it is best to ...”？

“Chinese writing system”译成“中文书写系统”比较好？

File: README.md [hunk 2]
```diff
@@ -18,28 +18,42 @@ Topics for discussion are suggested by [the gap-analysis template](https://w3c.g
 
 ### Documents 文档
 
-- [Chinese Layout Gap Analysis](https://w3c.github.io/clreq/gap-analysis/)
+- [Chinese Layout Gap Analysis | 中文支持差距分析](https://w3c.github.io/clreq/gap-analysis/)
 - [Requirements for Chinese Text Layout | 中文排版需求 (ED)](https://w3c.github.io/clreq/) ([WD](https://www.w3.org/TR/clreq/))
 
 ### Following 跟踪
 
 Rather than just 'Watch' this repository, it is best to subscribe to the [public-i18n-cjk](https://lists.w3.org/Archives/Public/public-i18n-cjk/) mailing list. That list is notified once a day (in digest form) about not only changes to this repository, but also about other W3C Working Group issues related to the Chinese writing systems. (Please use github issues rather than the mailing list to send feedback. The main purpose of the public-i18n-mongolian mailing list is for notifications; meeting minutes are also sent to public-i18n-cjk.)
```
Comment (upsuper) [inline README.md:26]: `public-i18n-mongolian` 这个写错了（

File: README.md [hunk 3]
```diff
@@ -18,28 +18,42 @@ Topics for discussion are suggested by [the gap-analysis template](https://w3c.g
...
+
 ### Contributing 贡献
 
 All contributors should read and agree with [CONTRIBUTING.md](https://github.com/w3c/clreq/blob/gh-pages/CONTRIBUTING.md).
 
+所有贡献者都应阅读并同意[CONTRIBUTING.md](https://github.com/w3c/clreq/blob/gh-pages/CONTRIBUTING.md)。
+
 Please use the [GitHub issue list](https://github.com/w3c/clreq/issues) to report issues for language support, for discussions, and to send feedback about documents.  (Learn [how GitHub issues work](http://w3c.github.io/i18n-activity/guidelines/issues.html).)
 
+请使用 [GitHub Issue](https://github.com/w3c/clreq/issues) 提出中文支持的问题、讨论以及对有关文档的反馈。 [学习如何使用GitHub（英文）](https://w3c.github.io/i18n-activity/guidelines/issues.html)
```
Comment (upsuper) [inline README.md]: 像英文那样“`（学习[如何使用GitHub（英文）](...)）`”会比较好？不然没头没脑地来这么一个子句有点莫名其妙……

File: README.md [hunk 4]
```diff
@@ -18,28 +18,42 @@ Topics for discussion are suggested by [the gap-analysis template](https://w3c.g
...
+
 **You may raise issues in Chinese, however any substantive discussions should be summarised in English at some point, so that non-Chinese speakers can follow the rationale and contribute comments.**
 
+**您可以用中文提出问题，但任何实质性讨论都应该用英语进行总结，以便非汉语使用者理解该问题并发表意见。**
```
Comment (upsuper) [inline README.md]: “however any substantive discussions should be summarised in English at some point” -> 但任何实质性讨论都应该**在某一阶段**用英文进行总结

File: README.md [hunk 5]
```diff
@@ -52,26 +66,19 @@ Chairs: 董福興 (Bobby TUNG), 陳奕鈞 (Yijun CHEN) • W3C staff: [Fuqiao Xu
...
+以下信息描述了 W3C 正在进行的 Web 语言支持方面的工作。
+
 - [Language matrix](https://w3c.github.io/typography/gap-analysis/language-matrix.html)
 - [Analysing support for text layout on the Web](https://github.com/w3c/i18n-discuss/wiki/Analysing-support-for-text-layout-on-the-Web)
 - [Overview of language enablement work in progress](https://www.w3.org/International/layout)
 - [Setting up a Gap Analysis Project](https://github.com/w3c/typography/wiki/Setting-up-a-Gap-Analysis-Project)
 - [Internationalization Sponsorship Program](https://www.w3.org/International/sponsorship/)
 
-### Links for editors
+### Links for editors 关于编辑文档
```
Comment (upsuper) [inline README.md]: 这个不译也罢？如果要译的话也应该是“文档编辑用”类似这种感觉……

File: README.md (hunk 1 above)
Comment (r12a) [reply README.md]: public-i18n-mongolian -> public-i18n-cjk

**Comment by xfq** [comment]
@upsuper Thanks for your review! I pushed some updates.

(re public-i18n-mongolian, it's fixed in https://github.com/w3c/clreq/commit/1bb20c0bb1f7b0302552320b9ce9aa3930257ade)

**Comment by xfq** [comment]
Merged.

**Pull request merged**